core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowExecutionsResource.scala

```diff
@@ -36,6 +37,45 @@ object WorkflowExecutionsResource {
       name: String
   )
 
+  /**
+    * This function retrieves the latest execution of a workflow
+    * @param wid
+    * @return WorkflowExecutionEntry
+    */
+  def getLatestExecution(wid: UInteger): Option[WorkflowExecutionEntry] = {
+    val executions = context
```

Comment:
no need for all these selection since we only need latest execution number

Reply:
refactored